packages/lib/server/repository/eventType.ts

```diff
@@ -814,6 +815,7 @@ export class EventTypeRepository {
         rrSegmentQueryValue: true,
         isRRWeightsEnabled: true,
         maxLeadThreshold: true,
+        includeNoShowInRRCalculation: true,
```

Comment:
Heads up @CarinaWolli the same logic that we use for determining the `luckyUser` is also used in `getAvailableSlots`. I added this as it was throwing a type error.